.travis.yml

```diff
@@ -92,33 +49,7 @@ jobs:
         - cd external/js/cothority
         - npm ci
         - npm link @dedis/kyber
-        - npm run test
-
-    - stage: deploy
-      name: "NPM: js > kyber"
-      <<: *stage_deploy_npm
-      deploy:
-        on:
-          branch: master
-        provider: script
-        script: >-
-          cd external/js/kyber &&
-          npm ci &&
-          npm version prerelease --preid=p`date +%y%m.%d%H.%M%S` &&
-          ./publish.sh --tag dev
-    - name: "NPM: js > cothority"
-      <<: *stage_deploy_npm
-      deploy:
-        on:
-          branch: master
-        provider: script
-        script: >-
-          cd external/js/kyber && npm ci && npm run link && cd ../../.. &&
-          cd external/js/cothority &&
-          npm ci &&
-          npm link @dedis/kyber &&
-          npm version prerelease --preid=p`date +%y%m.%d%H.%M%S` &&
-          ./publish.sh --tag dev
+        - npm run test:nocover -- --filter=SkipchainRPC
```

Comment:
Hum, maybe you didn't wanted to remove most of travis.

Reply:
yes, that's for testing. For the moment it fails testing, but only on Travis. So I just want to run that one test...

Reply:
This is why it's still in `WIP` ;)